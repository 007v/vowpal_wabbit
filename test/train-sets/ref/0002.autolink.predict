0.521144 PFF/20091028
0.521144 WIP/20091028
0.521144 GCC/20091028
0.521144 AAXJ/20091028
0.521144 VWO/20091028
0.268074 EEV/20091028
0.268074 GDX/20091028
0.411545 RTH/20091028
0.688761 MXI/20091028
0.482095 EWU/20091028
0.268074 SH/20091028
0.732510 EDC/20091028
0.268074 ERY/20091028
0.268074 SDS/20091028
0.285913 OEF/20091028
0.513833 IYT/20091028
0.268074 BIL/20091028
0.199231 GLL/20091028
0.142745 EDZ/20091028
0.732510 IWM/20091028
0.607247 VXF/20091028
0.574593 IJJ/20091028
0.527637 PIN/20091028
0.564661 XLB/20091028
0.445951 ECH/20091028
0.724819 TYH/20091028
0.527495 VAW/20091028
0.519525 DBP/20091028
0.703579 XME/20091028
0.674580 VO/20091028
0.651596 RSX/20091028
0.572246 EWC/20091028
0.629578 TUR/20091028
0.442918 VYM/20091028
0.575013 FCG/20091028
0.506632 VGT/20091028
0.546654 EWQ/20091028
0.544644 IEV/20091028
0.508010 XLK/20091028
0.561199 EFG/20091028
0.609416 BKF/20091028
0.578402 KIE/20091028
0.617598 EEB/20091028
0.602873 IJK/20091028
0.142745 DUG/20091028
0.142745 TWM/20091028
0.330844 MDY/20091028
0.477197 ACWI/20091028
0.158766 BSV/20091028
0.458853 DDM/20091028
0.572402 DIA/20091028
0.361757 TLT/20091028
0.142745 DXD/20091028
0.555503 XHB/20091028
0.481225 VDE/20091028
0.487424 BND/20091028
0.600417 EMB/20091028
0.484872 SCO/20091028
0.448172 AMJ/20091028
0.280663 OIL/20091028
0.610931 PZA/20091028
0.620368 VGK/20091028
0.626387 RWX/20091028
0.443647 JJA/20091028
0.646612 FXD/20091028
0.583555 XES/20091028
0.562398 VIG/20091028
0.198367 DZZ/20091028
0.589461 VFH/20091028
0.335243 DTO/20091028
0.546311 EWP/20091028
0.547099 FDN/20091028
0.541777 INP/20091028
0.229621 TYP/20091028
0.545686 RWR/20091028
0.583237 KBE/20091028
0.207156 EUO/20091028
0.470427 IWF/20091028
0.402676 SMN/20091028
0.589354 SMH/20091028
0.569477 XRT/20091028
0.348387 USO/20091028
0.493470 DJP/20091028
0.385700 CFT/20091028
0.237761 SRS/20091028
0.597423 MOO/20091028
0.479982 BIV/20091028
0.314014 VXX/20091028
0.623064 IYM/20091028
0.543805 IFN/20091028
0.618478 SLV/20091028
0.553395 TAO/20091028
0.605386 PGF/20091028
0.601763 IYR/20091028
0.254562 QID/20091028
0.561137 THD/20091028
0.571849 IJS/20091028
0.562749 VB/20091028
0.485643 EDV/20091028
0.512602 IEZ/20091028
