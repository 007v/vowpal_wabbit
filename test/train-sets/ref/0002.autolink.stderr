predictions = 0002.autolink.predict
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/0002.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
0.000000 0.000000            1            1.0   0.5211   0.5211       15
0.000100 0.000199            2            2.0   0.5353   0.5211       15
0.001288 0.002476            4            4.0   0.5854   0.5211       15
0.030920 0.060552            8            8.0   0.5575   0.4115       15
0.025017 0.019114           16           16.0   0.5878   0.5138       15
0.018755 0.012493           32           32.0   0.6038   0.5722       15
0.018303 0.017851           64           64.0   0.5683   0.4436       15

finished run
number of examples per pass = 100
passes used = 1
weighted example sum = 100.000000
weighted label sum = 52.222146
average loss = 0.013972
best constant = 0.522221
total feature number = 1499
